Add explainer for "Expose initiator info for resourceTiming"

guohuideng2024 opened the pull request:
Move the explainer into MS repository.

Also fixed a few white spaces.

dandclark reviewed (approved):
Thanks for moving this over!

I suggested a few typo fixes, and from the bot comment in the PR it looks like you need to post a reply to agree to the CLA. After that I can merge.

Andi it's not blocking for this PR, but as a follow-up can you please do Steps 3.2 and 3.3 from [Launching a new feature or significant code change - Overview](https://microsoft.visualstudio.com/Edge/_wiki/wikis/Edge.wiki/849/Launching-a-new-feature-or-significant-code-change?anchor=publishing-an-explainer-(from-private-to-public)) to create: a) new issue label b) new issue template c) edit to README.md.
Thanks!

guohuideng2024 commented:
@microsoft-github-policy-service agree company="Microsoft"

guohuideng2024 commented:
> Thanks for moving this over!
> 
> I suggested a few typo fixes, and from the bot comment in the PR it looks like you need to post a reply to agree to the CLA. After that I can merge.
> 
> Andi it's not blocking for this PR, but as a follow-up can you please do Steps 3.2 and 3.3 from [Launching a new feature or significant code change - Overview](https://microsoft.visualstudio.com/Edge/_wiki/wikis/Edge.wiki/849/Launching-a-new-feature-or-significant-code-change?anchor=publishing-an-explainer-(from-private-to-public)) to create: a) new issue label b) new issue template c) edit to README.md. Thanks!

Thank you for reviewing! The new commit should have addressed all the errors you pointed out. I added my github account info too and yes it's `Timing-Allow-Origin` :). 

I added a reply hoping it would fix the CLA issue. I will do the other steps you mentioned shortly (within this week).

I made additional changes. I found out I didn't completely update my "security/privacy" section the last time I edit this file, and I should have added a detailed discussion on cached resources.

Guohui
